packages/builder/src/helpers/data/utils.js

```diff
@@ -119,3 +119,73 @@ export function flipHeaderState(headersActivity) {
   })
   return enabled
 }
+
+// convert dynamic variables list to simple key/val object
+export function getDynamicVariables(datasource, queryId) {
```

Comment:
This is fair enough, I spoke to cheeks about this and we can't actually put functions under the `pages` directory in a JS file as it has risk of confusing routify, it uses the structure of the file system within that directory as the routing controls for the builder app - any file within will be considered as part of the routes although I'm not sure if it would ignore JS files.

I'd originally broken them out as I found the REST interface svelte file was getting very out of control, the script section contained a lot of utility functions that didn't really need to be seen when working on it so breaking them out somewhere else just made the whole thing easier to look at - I had originally wanted to go with the approach of putting them in something like `rest/[query]/_utils.js` but was concerned it would cause problems for routify as there should only be svelte files in those directories - after a discussion it was thought best to put them in the helpers directory.